<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Warframe Extractor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTreeView" name="treeView"/>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="Preview">
       <attribute name="title">
        <string>Preview</string>
       </attribute>
       <widget class="QOpenGLWidget" name="openGLWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>300</width>
          <height>200</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Details">
       <attribute name="title">
        <string>Metadata</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>100</y>
          <width>291</width>
          <height>231</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="FullPathContainer">
           <item>
            <widget class="QLabel" name="FullPathLabel">
             <property name="text">
              <string>Full Path</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="FullPathData">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="TimestampContainer">
           <item>
            <widget class="QLabel" name="TimestampLabel">
             <property name="text">
              <string>Timestamp</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="TimestampData">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="CompressedLengthContainer">
           <item>
            <widget class="QLabel" name="CompressedLengthLabel">
             <property name="text">
              <string>Compressed Length</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="CompressedLengthData">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
